spec/models/user_spec.rb

```diff
@@ -18,26 +18,26 @@
   end
 
   describe '#full_name' do
-      let(:first_name) { Faker::Name.neutral_first_name }
-      let(:last_name) { Faker::Name.last_name }
+    let(:first_name) { Faker::Name.neutral_first_name }
+    let(:last_name) { Faker::Name.last_name }
 
     it 'returns the full name of the user' do
-      user = User.new(first_name: first_name, last_name: last_name)
+      user = described_class.new(first_name: first_name, last_name: last_name)
```

Comment:
Recorda que las variables en los tests se definen usando `let`, y fuera de los bloques `it`. Puede que a esta branch le falte nivelación